File: Python/request.py
# imports
import aiohttp
import asyncio

# your api token
kawaii_token = "token"

# request
async def kawaii(main, sub, filter):
    async with aiohttp.ClientSession() as session:
        async with session.get(f'https://kawaii.red/api/{main}/{sub}/token={kawaii_token}&filter={filter}/') as r:
            js = await r.json()
            return str(js["response"])

# print async environment (normally standard in bot environment)
async def main():
    print(await kawaii("gif", "kiss", []))

# create async environment
loop = asyncio.get_event_loop()
loop.run_until_complete(main())

File: JavaScript/request.js
const { Kawaii } = require("kawaii-api");

const api = new Kawaii("token");

api.get("gif", "kiss").then((result) => {
        console.log(result);
    });

api.endpoints("gif").then((result) => {
        console.log(result);
    });

// or async/await

const get = async() => {
    const result = await api.get("gif", "kiss");
    console.log(result);
}

const endpoints = async() => {
    const result = await api.endpoints("gif");
    console.log(result);
}